### PR TITLE
Fixed links in rst files

### PR DESCRIPTION
**Pull request by klmcadams**
Hi @jorgepiloto, I have a few questions:

In [doc/source/how-to/dns-configuration.rst](https://github.com/ansys/pyansys-dev-guide/compare/doc/fix-links?expand=1#diff-a1ac5cc6232feb6e57b5695a66998392f172cf00b93462750087516eece4ed4b), should pyansys.com be docs.pyansys.com? 

Here are some other links I couldn't find:

In dev.docs.pyansys.com/getting-started/basic.html:
system-reporting-tool: https://github.com/ansys/system-reporting-tool
	- Is this the system reporting tool? [ansys/pyansys-tools-report](https://github.com/ansys/pyansys-tools-report)

In dev.docs.pyansys.com/how-to/documenting.html:
actions.docs.pyansys.com is not a secure website/it says there isn't a GitHub Page site, so I couldn't update these links below

- pyansys/actions/doc-deploy-stable action:  https://actions.docs.pyansys.com/version/stable/doc-actions/index.html#doc-deploy-stable-action
- pyansys/actions@v4:  https://actions.docs.pyansys.com/version/stable/index.html - page not found

### CODE REVIEW

**Comment by RobPasMue** [comment]
Hi @klmcadams - couple of points:

* I think the references to ``pyansys.com`` alone were fine, we were just trying to point out that the ``pyansys.com`` domain is under our control. Could you revert those back?
* Please change the system-reporting-tool to the pyansys-tools-report one
* The actions domain was modified recently. It is now ``actions.docs.ansys.com``. Good catch!

**Comment by RobPasMue** [comment]
By the way...

I implemented something on the ``pyansys`` metapackage that could be useful, at least for this package @jorgepiloto. 

See https://github.com/ansys/pyansys/blob/ca64552ea84566a873e29735ee5c4574074b1428/doc/Makefile#L21-L25

By overriding the ``html`` target, I am forcing ``make`` to run the ``linkcheck`` target first. This ``linkcheck`` what it does is verify that all URLs in our documentation are working fine. We should consider implementing it here (and maybe in our actions?). What do you think @klmcadams @jorgepiloto? Automation is the way to go so if we can have our workflows tell us which links are broken, even better!

File: doc/source/getting-started/basic.rst [hunk 1]
```diff
@@ -7,16 +7,16 @@ libraries that interface with Ansys products or services. To try out a
...
+* `PyDPF-Post <https://github.com/ansys/pydpf-post>`_
 * `PyMAPDL`_
 * `PyMAPDL Legacy Reader <https://github.com/ansys/pymapdl-reader>`_
 * `PyPIM <https://github.com/ansys/pypim>`_
 
 If you want to create, develop, or contribute to a PyAnsys library, 
 visit these links:
 
-* `PyAnsys Developer's Guide <https://github.com/ansys/about>`_
+* `PyAnsys Developer's Guide <https://dev.docs.pyansys.com/>`_
```
Comment (jorgepiloto) [inline doc/source/getting-started/basic.rst:19]: Knowing that the lines below this one point to a GitHub repo, let us do the same with this one:

```suggestion
* `PyAnsys Developer's Guide <https://github.com/ansys/pyansys-dev-guide>`_
```

File: doc/source/how-to/dns-configuration.rst [hunk 2]
```diff
@@ -6,9 +6,9 @@ online under the following canonical name (CNAME) convention:
 
 ``https://<product>.docs.pyansys.com``
 
-To request a CNAME for the ``pyansys.com`` domain, contact
-`Maxime Rey`_, `Roberto Pastor Muela`_ or `Alex Kaszynski`_. Any of these members
-can handle the creation of the requested PyAnsys subdomain.
+To request a CNAME for the ``docs.pyansys.com`` domain, contact the
```
Comment (jorgepiloto) [inline doc/source/how-to/dns-configuration.rst:9]: The domain is ``pyansys.com`` and ``docs.pyansys.com`` is a subdomain. Although we usually create ``*.docs.pyansys.com``, let's leave the ``pyansys.com`` reference in here.

```suggestion
To request a CNAME for the ``pyansys.com`` domain, contact the
```

File: doc/source/how-to/dns-configuration.rst [hunk 3]
```diff
@@ -17,24 +17,22 @@ for your documentation, refer to `Managing a custom domain for your GitHub Pages
 DNS TXT verification
 --------------------
 
-Once a CNAME is registered under the ``pyansys.com`` domain, the next step is
+Once a CNAME is registered under the ``docs.pyansys.com`` domain, the next step is
```
Comment (jorgepiloto) [inline doc/source/how-to/dns-configuration.rst:20]:
```suggestion
Once a CNAME is registered under the ``pyansys.com`` domain, the next step is
```

File: doc/source/how-to/dns-configuration.rst [hunk 4]
```diff
@@ -17,24 +17,22 @@ for your documentation, refer to `Managing a custom domain for your GitHub Pages
...
 IT department to provide a DNS TXT verification. To verify a new CNAME for an
 organization, refer to `Verifying a domain for your organization site`_. This guide
 shows how to create DNS TXT verification elements for GitHub Pages sites.
 
 .. warning::
 
-    Only users with privilege access to the ``pyansys.com`` DNS zone can
-    perform this operation. Contact `Maxime Rey`_, `Roberto Pastor Muela`_
-    or `Alex Kaszynski`_ if needed.
+    Only users with privilege access to the ``docs.pyansys.com`` DNS zone can
```
Comment (jorgepiloto) [inline doc/source/how-to/dns-configuration.rst:28]:
```suggestion
    Only users with privilege access to the ``pyansys.com`` DNS zone can
```

File: doc/source/how-to/dns-configuration.rst [hunk 5]
```diff
@@ -17,24 +17,22 @@ for your documentation, refer to `Managing a custom domain for your GitHub Pages
...
+    perform this operation. Contact the `PyAnsys Core Team`_ if needed.
 
 PyAnsys verified domains
 ------------------------
 
-In the `PyAnsys GitHub organization`_, these domains have been verified:
+In the `PyAnsys GitHub organization`_, this domain has been verified:
 
-* ``pyansys.com``
 * ``docs.pyansys.com``
```
Comment (jorgepiloto) [inline doc/source/how-to/dns-configuration.rst:36]:
```suggestion
* ``pyansys.com``
* ``docs.pyansys.com``
```

File: doc/source/how-to/dns-configuration.rst [hunk 6]
```diff
@@ -17,24 +17,22 @@ for your documentation, refer to `Managing a custom domain for your GitHub Pages
...
-    perform this operation. Contact `Maxime Rey`_, `Roberto Pastor Muela`_
-    or `Alex Kaszynski`_ if needed.
+    Only users with privilege access to the ``docs.pyansys.com`` DNS zone can
+    perform this operation. Contact the `PyAnsys Core Team`_ if needed.
 
 PyAnsys verified domains
 ------------------------
 
-In the `PyAnsys GitHub organization`_, these domains have been verified:
+In the `PyAnsys GitHub organization`_, this domain has been verified:
```
Comment (jorgepiloto) [inline doc/source/how-to/dns-configuration.rst:34]:
```suggestion
In the `PyAnsys GitHub organization`_, these domains have been verified:
```

**Review by jorgepiloto — CHANGES_REQUESTED** [review]
Thanks for adding this, @klmcadams. I just left some minor suggestions regarding the domains.

**Comment by jorgepiloto** [comment]
Thanks for remembering us the tool, @RobPasMue. I told @klmcadams we used to have a tool for checking the integrity of the links. Let incorporate the changes suggested by Roberto.

**Comment by RobPasMue** [comment]
> Thanks for remembering us the tool, @RobPasMue. I told @klmcadams we used to have a tool for checking the integrity of the links. Let incorporate the changes suggested by Roberto.

I'll open an issue for this. Thank you both!

**Comment by jorgepiloto** [comment]
Regarding the security of the actions, it looks like when we migrated, I added the "ansys.github.io/actions" while IT created the actual domain. Later, I forgot about this situation. Just merged [this PR](https://github.com/ansys/actions/actions/runs/5342314925) to fix this situation.

**Review by RobPasMue — APPROVED** [review]
LGTM! Thanks @klmcadams

**Comment by klmcadams** [comment]
> LGTM! Thanks @klmcadams

You're welcome!

**Review by jorgepiloto — APPROVED** [review]

**Pull request merged**